src/components/Sweep/Sweep.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+import { Typography, Form } from 'antd';
+
+import { SCREEN_BREAKPOINT } from '../../constants';
+import SweepExecute from './SweepExecute';
+import SweepApprove from './SweepApprove';
+
+function Sweep() {
+  return (
+    <Container>
+      <Form name="basic" labelCol={{ span: 6 }} initialValues={{ remember: true }} autoComplete="on" size="large">
```

Comment:
- Please change the form name to something more appropriate
- Where is `remember: true` documented? I did not see that in Antd docs. Also how does it work? Does it preserve the values in localStorage?

Reply:
This is not enough. The form values must be persisted and restored automatically as per the requirements. You can use `zustand` for that, see e.g. https://github.com/chronologic/automate-ui/blob/main/src/components/BatchTx/useBatchConfigStorage.ts#L27